openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -193,8 +170,11 @@ const GlossaryPageV1 = () => {
           extend(treeNode, { ...data, children });
 
           setSelectedData(data);
-          if (key) {
-            handleSelectedKey(key);
+          if (fqn) {
```

Comment:
Can we have some comments to explain what is achieved here?

Reply:
Done